File: src/targets/node/axios/fixtures/multipart-form-data-no-params.js
<<<<<<< HEAD
const axios = require('axios');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'Content-Type': 'multipart/form-data'}
};

axios
  .request(options)
  .then(function (response) {
    console.log(response.data);
  })
  .catch(function (error) {
    console.error(error);
  });
=======
const axios = require('axios').default;

const options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  headers: {'Content-Type': 'multipart/form-data'}
};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}
>>>>>>> upstream/master

File: src/targets/java/nethttp/fixtures/multipart-form-data-no-params.java
HttpRequest request = HttpRequest.newBuilder()
<<<<<<< HEAD
    .uri(URI.create("https://httpbin.org/anything"))
=======
    .uri(URI.create("http://mockbin.com/har"))
>>>>>>> upstream/master
    .header("Content-Type", "multipart/form-data")
    .method("POST", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());